.sync.yml

```diff
@@ -3,4 +3,5 @@
   secure: "fH1bR1eBgA9tztMHtgdBSEOhNChC2YyoAczMzjUWgfDjinU1WOUPEprfV9upf6ZeQSTxAES36BBOrdxB6BVgdXlvLlCID+Lge1fTfi75ZX34q+eTadig/iZp3M0OveL2b5OmDv0cGbmMq8ywhgG+GPFuwPy7HF1Qjgul8UGNFOs="
   docker_sets:
     - set: centos7-64
-    - set: ubuntu1404-64
+spec/spec_helper_acceptance.rb:
+  unmanaged: true
```

Comment:
Isn't this the default?

Reply:
Yes, but I wanted to make sure that this setting has not been forgotten. Otherwise someone could set it to false and break things.

Reply:
You'd see that on the next modulesync and it'd fail. I wouldn't copy it.

Reply:
Alright. I hope you are right. ;)